mmv1/templates/terraform/operation_retry/service_usage.go

```diff
@@ -0,0 +1,11 @@
+// Retries errors on 403 3 times if the error message
```

Comment:
Won't this just get reset on every iteration? I don't think the code allows us to retry `n` times as written (and the change to do so would be messy- not sure we want to do it to the common op code)

Reply:
umm I'm confused? This is specific to three times. We use `w.retryCount` to monitor the count.

If you look at the downstream code it should make more sense.